internal/apis/certmanager/validation/issuer.go

```diff
@@ -404,10 +404,6 @@ func ValidateACMEChallengeSolverDNS01(p *cmacme.ACMEChallengeSolverDNS01, fldPat
 			if len(p.Route53.Region) == 0 {
 				el = append(el, field.Required(fldPath.Child("route53", "region"), ""))
 			}
-			// accessKeyID or accessKeyIDSecretRef must be specified, but not both
-			if len(p.Route53.AccessKeyID) == 0 && p.Route53.SecretAccessKeyID == nil {
-				el = append(el, field.Required(fldPath.Child("route53"), "accessKeyID or accessKeyIDSecretRef is required"))
-			}
```

Comment:
nitpick (non-blocking): It'd be nice to have a comment here explaining why leaving both out would be valid / desired in some cases, but definitely not required!

Reply:
Thanks, yeah let's include that